components/news/Home.vue

```diff
@@ -0,0 +1,115 @@
+<template>
+  <div class="w-full">
+    <nuxt-link :to="'/news/' + news.slug">
+      <div class="news-image">
+        <img :src="newsImage" :alt="news.title.rendered" class="">
+      </div>
+    </nuxt-link>
+    <div class="grid news-infos">
+      <div class="news-date">
+        <div class="newsgeneral">
+          <nuxt-link to="news/">
+            <h3 class="m-0">
+              News
+            </h3>
+          </nuxt-link>
+        </div>
+        <div class="newsdate">
+          <p>{{ format(new Date(news.modified), 'MM') }}</p>
+          <p>{{ format(new Date(news.modified), 'dd') }}</p>
+        </div>
+      </div>
+      <div class="px-4 news-info-details">
+        <h5 class="py-4">
+          <nuxt-link :to="'/news/' + news.slug">
+            {{ htmlDecoder(news.title.rendered) }}
+          </nuxt-link>
+        </h5>
+        <div v-html="news.excerpt.rendered" />
+        <div class="flex justify-end my-4">
+          <div class="px-4 py-2 bg-gray-200 rounded-lg cursor-pointer more-button" @click="$emit('changenews')">
```

Comment:
Well I was never fan and never understood the js stuff but I have to say these click events and this emit stuff is really really smooth & cool. Good job!

Reply:
$emit emits from children to parent component